tests/rkt-monitor/main.go

```diff
@@ -149,11 +154,14 @@ func runRktMonitor(cmd *cobra.Command, args []string) {
 		time.Sleep(time.Second)
 	}
 
+	loadAvg, err := load.Avg()
+
+	containerStoping := time.Now()
```

Comment:
also Stoping -> Stopping

do you just want to close this PR in favour of your other one?